.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- 
+
+Thank you for submitting an issue!
+
+If you are reporting a bug, please provide detailed reproduction instructions, including:
+
+- How you installed `codex`
+- What your operating system is
+- What sort of project you're running - old cabal, stack, cabal new-build, etc.
+- Exactly the output you recieve from `codex`
+- What output you expect to see
```

Comment:
May want to add, "what command you used to invoke `codex` including what arguments you passed"

Reply:
Good call, I've added that note.